### PR TITLE
Quiznos

### PR DESCRIPTION
**Pull request by MagnetonIO**
Added a quiznos spider. Please check.

### CODE REVIEW

File: locations/spiders/quiznos.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    start_urls = (
+        'http://restaurants.quiznos.com/data/stores.json?callback=storeList',
+    )
+
+    def parse(self, response):
+
+        try:
+            data = response.body_as_unicode()
+            stores = json.loads(re.search('storeList\((.*)\)', data).group(1))
+        except:
```
Comment (iandees) [inline locations/spiders/quiznos.py]: Please don't catch exceptions like this. I'd prefer to have them bubble up and cause the spider to stop.

Comment (MagnetonIO) [reply locations/spiders/quiznos.py]: Don't you need statements for validation?

Comment (iandees) [reply locations/spiders/quiznos.py]: Not in this case. I'd rather the spider throw an exception and have it get logged through the normal scrapy logging system than have it get caught and logged separately.

**Comment by MagnetonIO** [comment]
I updated quiznos for exception. issue. Can you check again?

File: locations/spiders/quiznos.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+                lon=store.get('longitude'),
+                ref=str(store.get('storeid')),
+                phone=store.get('phone'),
+                name=store.get('restaurantname'),
+                opening_hours=store.get('businesshours'),
+                addr_full=store.get('address1'),
+                city=store.get('city'),
+                state=store.get('statecode'),
+                postcode=store.get('zipcode'),
+                website=store.get('url'),
```
Comment (iandees) [inline locations/spiders/quiznos.py]: The website is not a fully formed URL. Looks like you need to prepend something to the front of the value from `url`.

File: locations/spiders/quiznos.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+        for store in stores:
+
+            yield GeojsonPointItem(
+                lat=store.get('latitude'),
+                lon=store.get('longitude'),
+                ref=str(store.get('storeid')),
+                phone=store.get('phone'),
+                name=store.get('restaurantname'),
+                opening_hours=store.get('businesshours'),
```
Comment (iandees) [inline locations/spiders/quiznos.py]: The hours format from the website doesn't match our desired hours format. Take a look at other scrapers to see how they parse the various hours formats into something consistent.

**Pull request merged**